app/controllers/api/v1/admins_controller.rb

```diff
@@ -0,0 +1,48 @@
+class AdminsController < ApplicationController
```

Comment:
also, we need to change the name:

since we are nested underneath /api Rails is going to need to see that in the class name.

```Api::V1::AdminsController```